google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryOptions.java

```diff
@@ -81,6 +93,19 @@ protected BigQueryRpcFactory getDefaultRpcFactory() {
   }
 
   @Override
+  public TransportOptions getDefaultTransportOptions() {
+    return getDefaultHttpTransportOptions();
+  }
+
+  public static HttpTransportOptions getDefaultHttpTransportOptions() {
+    return HttpTransportOptions.newBuilder().build();
+  }
+
+  public HttpTransportOptions getHttpTransportOptions() {
```

Comment:
Ahh of course, use a covariant return type. I'll do that.

(leaving the discussion of HttpServiceOptions to the comment thread above)